lib/ansible/modules/dnf5.py

```diff
@@ -833,7 +821,11 @@ def main():
             auto_install_module_deps=dict(type="bool", default=True),
         )
     )
-    Dnf5Module(AnsibleModule(**yumdnf_argument_spec)).run()
+    module = AnsibleModule(**yumdnf_argument_spec)
+    try:
+        Dnf5Module(module).run()
+    except LIBDNF5_ERRORS as e:
+        module.fail_json(msg=str(e), failures=[], rc=1)
```

Comment:
Is there a reason to include `failures=[], rc=1` here?

Reply:
This is still a backwards compatibility with the `yum` module that historically returned those keys. Users can still use `- yum:` and be redirected to the `dnf5` module through the `dnf` action plugin.